src/core/game.vala

```diff
@@ -3,4 +3,6 @@
 private interface Games.Game : Object {
 	public abstract string name { get; }
 	public abstract Icon? icon { get; }
+
+	public abstract Runner get_runner ();
```

Comment:
[Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter) ?

What about passing the runner to the constructor, and have a `Game.run()` method which delegates to the runner?